### PR TITLE
feat: migrated cpms/cpms-payment-test to mot-cpms-payment-test

### PR DESCRIPTION
**Pull request by jennifer-choo**
## Description
Migration of cpms payment test repository from Gitlab to Github

Related issue: [BL-14418](https://dvsa.atlassian.net/browse/BL-14418)

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

**Review by faggc — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Include a summary of the change here.
+-->
+
+Related issue: [JIRA_TICKET_NUMBER](LINK_TO_JIRA_TICKET)
+
+## Before submitting (or marking as "ready for review")
+
++ [ ] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
++ [ ] Have you performed a self-review of the code
++ [ ] Have you have added tests that prove the fix or feature is effective and working
```
Comment (caoimheb) [inline .github/pull_request_template.md]:
```suggestion
+ [ ] Have you performed a self-review of the code?
+ [ ] Have you have added tests that prove the fix or feature is effective and working?
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  cancel-in-progress: true
+
+jobs:
+  static:
+    uses: dvsa/.github/.github/workflows/php-library-static.yml@v3.2.3
+
+  security:
+    uses: dvsa/.github/.github/workflows/php-library-security.yml@v3.2.3
+    with:
+      php-versions: "[\"8.0\", \"8.1\"]"
```
Comment (caoimheb) [inline .github/workflows/ci.yml]: `composer.json` also has `7.3`. Probably worth adding here too

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  cancel-in-progress: true
+
+jobs:
+  static:
+    uses: dvsa/.github/.github/workflows/php-library-static.yml@v3.2.3
+
+  security:
+    uses: dvsa/.github/.github/workflows/php-library-security.yml@v3.2.3
+    with:
+      php-versions: "[\"7.3\",\"8.0\", \"8.1\"]"
```
Comment (caoimheb) [inline .github/workflows/ci.yml]: Looks like here and in composer files will need bumped to 7.4 to sort failing Github actions

**Review by caoimheb — COMMENTED** [review]
There are some vulnerabilities being flagged by the snyk actions. Worth seeing which ones are failing and if they are easily fixed. If not, we'll need to raise another ticket to resolve them.

**Comment by caoimheb** [comment]
> There are some vulnerabilities being flagged by the snyk actions. Worth seeing which ones are failing and if they are easily fixed. If not, we'll need to raise another ticket to resolve them.

Nevermind, I see you already have ticket created 😄

**Review by caoimheb — APPROVED** [review]

**Comment by jennifer-choo** [comment]
> > There are some vulnerabilities being flagged by the snyk actions. Worth seeing which ones are failing and if they are easily fixed. If not, we'll need to raise another ticket to resolve them.
> 
> Nevermind, I see you already have ticket created 😄

Yeah, was thinking of updating them myself on this ticket, but because these are abstract test classes used by other cpms repo unit tests, it would be difficult to check if anything broke in the process so I made a separate ticket

**Pull request merged**